portal-web/test/functional/com/liferay/portalweb/paths/portlet/claysample/ClaySamplePortlet.path

```diff
@@ -106,6 +106,17 @@
 	<td>//section[contains(@id,'portlet_ClaySamplePortlet')]</td>
 	<td></td>
 </tr>
+<!--MANAGEMENT_TOOLBARS-->
+<tr>
+	<td>DISPLAY_CONTEXT_ANY_ICON</td>
+	<td>//h3[text()='USING DISPLAY CONTEXT']//following::nav[contains(@class,'management-bar')]//*[*[name()='svg'][contains(@class,'icon-${key_text}')]]</td>
```

Comment:
`//h3[text()='USING DISPLAY CONTEXT']//following::nav[contains(@class,'management-bar')][1]//*[*[name()='svg'][contains(@class,'icon-${key_text}')]]`

I would insert a [1] b/c there's a possibility of other nav elements that follow after USING DISPLAY CONTEXT text based on how the toolbars are structured in sample portlet.